csrc/include/natten/cuda/fna/na_utils.cuh

```diff
@@ -410,6 +410,25 @@ inline NA3dDim tuple_to_na_dim(std::tuple<int32_t, int32_t, int32_t> v) {
   return NA3dDim(std::get<0>(v), std::get<1>(v), std::get<2>(v));
 }
 
+template <typename BoolTupleType>
+bool bool_tuple_or(BoolTupleType tuple);
```

Comment:
```suggestion
bool tuple_or(BoolTupleType tuple);
```
Nit: logical operators like or mostly make sense for boolean types, and in this case, only causal mask.